packages/react/src/components/Handle/index.tsx

```diff
@@ -159,6 +159,7 @@ function HandleComponent(
       isValidConnection: isValidConnectionStore,
       lib,
       rfId: flowId,
+      nodeLookup,
```

Comment:
I think we would need the same changes for Svelte Flow too?

Reply:
This is just used in context of click connect which is not present in Svelte Flow